### ISSUE TITLE
compile dothtml pages

### ISSUE THREAD

**Issue by ctisolutionspl** [riganti/dotvvm#986]
Hello,
Is it possible to precompile dothtml and dotcontrol files in a similar way to using RazorGenerator in MVC projects?
I dont want to include these files in distribution package. In standard mvc app it was possible to compile it inside DLL file using RazorGenerator.

**Comment by exyi** [comment]
Full precompilation is not currently supported.

You can however embed the pages into the resulting assembly as embedded resource, so you don't have to copy the Views folder. See https://www.dotvvm.com/docs/tutorials/control-development-markup-control-registration/2.0 - we have it documented for controls, but it works exactly the same for Views

**Comment by ctisolutionspl** [comment]
do you have plans to support precompilation?

**Comment by exyi** [comment]
Yes, but the only thing we can promise that it's not going to happen very soon as it's not that easy and there is not that many benefits.

**Comment by MarekPokornyOva** [comment]
I tried some investigation on this field as fun and you can see the result on https://github.com/MarekPokornyOva/SourceGeneratorForDotvvmViews. Please don't have high expectations.
